### PR TITLE
Closes #1 - Establish some base patterns and conventions

### LINKED ISSUE #1: Establish patterns and conventions around the library
Simply put: Start scaffolding out the library, establish the base patterns for the hooks, management, context, etc.

### PR DESCRIPTION
**Pull request by rblalock**
Addresses #1

### CODE REVIEW

File: example/src/blockTypes/HTMLBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { IBlock } from '@pinpt/block-editor/dist/interfaces';
```
Comment (chrisbowley) [inline example/src/blockTypes/HTMLBlock.tsx:2]: 🤔

Comment (rblalock) [reply example/src/blockTypes/HTMLBlock.tsx:2]: LOL.  thx.

Comment (rblalock) [reply example/src/blockTypes/HTMLBlock.tsx:2]: What are you contemplating?

Comment (chrisbowley) [reply example/src/blockTypes/HTMLBlock.tsx:2]: Oh this is the example app... as you were

**Review by chrisbowley — APPROVED** [review]

**Pull request merged**